shared/protocol/src/main/java/io/pravega/shared/protocol/netty/CommandEncoder.java

```diff
@@ -59,6 +62,12 @@
  * need to be parsed out of individual messages. Notably this includes the event number of the last
  * event in the block, so that it can be acknowledged.
  *
+ * if the channel is not free, then each append is enqueued to the session's pending List.
+ * This session pending list gets flushed under following conditions
```

Comment:
Are these conditions "AND" or "OR"? In other words, must all of them be true or does at least one be true?

Reply:
any one condition; Updated documentation accordingly.